frontend/src/metabase/lib/pulse.ts

```diff
@@ -79,15 +86,15 @@ export function fieldsAreValid(channel, channelSpec) {
     .every(field => Boolean(channel.details?.[field.name]));
 }
 
-function pulseChannelsAreValid(pulse, channelSpecs) {
+function pulseChannelsAreValid(pulse: Pulse, channelSpecs: any) {
```

Comment:
That `any` there is an admission of ignorance on my part. 

If anyone knows about it in detail, we can make it more specific.